src/connection_config.rs

```diff
@@ -1,10 +1,10 @@
-use std::cell::RefCell;
-use std::sync::OnceLock;
+use glib::subclass::Signal;
+use glib::Binding;
 use gtk::prelude::*;
 use gtk::subclass::prelude::*;
-use gtk::{gio, glib, CompositeTemplate, Entry, Button};
-use glib::Binding;
-use glib::subclass::Signal;
+use gtk::{gio, glib, Button, CompositeTemplate, Entry};
```

Comment:
I am new in Rust, but does the order really matter? The same in the other files.

Reply:
Functionally the order does not matter, but alphabetical order is the preferred order upheld by rustfmt.

Reply:
Removed the fmt and clippy fix changes altogether from this PR, will make a separate PR for those.